Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -432,10 +432,10 @@ private bool CheckForCollisionBetween()
                 return false;
             }
 
-            if (DebugViewSettings.drawInterceptChecks)
+            /*if (DebugViewSettings.drawInterceptChecks)
```

Comment:
Why are these commented out?

Reply:
The value "drawInterceptChecks" of "DebugViewSettings" does not exist anymore,
and I was unable to find anything similar. Unless a custom debug setting/button is planned I don't know how you would trigger it ingame.

Reply:
It does correspond to that button. If it got moved to something else these calls should be moved accordingly. If it was removed then they need to move to the CE debug menu.

Reply:
Replaced it with Controller.settings.DebugDrawInterceptChecks, and created variables accordingly.